scholia/app/templates/clinical_trial.html

```diff
@@ -157,12 +20,12 @@ <h1 id="h1">Clinical trial</h1>
 
 <h2>Data</h2>
```

Comment:
@ammar257ammar, the original pattern used by Finn also has IDs on the header elements, with the ID name like `foo-header` for `foo-table`. So, here you'd have `id="data.header"`.

Reply:
Wouldn't it be `id="data-header"`?

Reply:
@ammar257ammar I have added a few other places where the old "ids" are wrong.